content/cli/snapshots.md

```diff
@@ -11,7 +11,7 @@ The `cfy snapshots` command is used to manage data snapshots of Cloudify manager
 
 You can use the command to create, upload, download, delete and list snapshots and also to restore a Manager using a snapshot archive.
 
-See [snapshots]({{< relref "manager/snapshots.md" >}}) for more information.
+For more about working with snapshots, go to: [snapshots]({{< relref "manager/snapshots.md" >}}).
```

Comment:
This has the manager/snapshots.md path, but you closed the PR which actually made that file accurate so there's no point linking to the broken one.